omero/users/cli/chown.rst

```diff
@@ -135,7 +135,7 @@ would be transferred to user 10.
 Including and excluding objects
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. program:: omero chown
```

Comment:
Is this correct?  (Note `chown` is a system command so `omero chown` is much less ambiguous).